pyccel/ast/utilities.py

```diff
@@ -172,69 +172,57 @@ def builtin_function(expr, args=None):
 
     return None
 
+
 # TODO add documentation
-builtin_import_registery = ('numpy', 'numpy.linalg', 'numpy.random', 'scipy.constants', 'itertools', 'math', 'pyccel.decorators')
+builtin_import_registery = {'numpy': {**numpy_functions, **numpy_constants, 'linalg':numpy_linalg_functions, 'random':numpy_random_functions},
+        'numpy.linalg': numpy_linalg_functions,
+        'numpy.random': numpy_random_functions,
```

Comment:
Aren't these, in practice, a repetition of the keys below?
```
builtin_import_registery['numpy']['linalg']
builtin_import_registery['numpy']['random']
```

Reply:
Not quite. This handles the syntax
```
import numpy

numpy.linalg.func
```

Whereas the standard method handles 
```
import numpy.linalg as mod 
```

Maybe we could handle this differently to avoid repetition

Reply:
I see. This is the fine distinction between (1) importing a sub-module and (2) getting its attributes later. The repetition is very little for now, but this may need to be improved in the future.